ReactSkia/components/RSkComponentView.cpp

```diff
@@ -14,19 +14,16 @@ RSkComponentView::RSkComponentView(const ShadowView &shadowView)
 void RSkComponentView::OnPaint(SkCanvas *canvas) {
   auto component = getComponentData();
   auto const &viewProps = *std::static_pointer_cast<ViewProps const>(component.props);
-  auto bgcolor = colorComponentsFromColor(viewProps.backgroundColor);
-  float ratio = 255.9999;
-  SkPaint paint;
-  paint.setColor(SkColorSetARGB(
-      bgcolor.alpha * ratio,
-      bgcolor.red * ratio,
-      bgcolor.green * ratio,
-      bgcolor.blue * ratio));
-
-  auto framePoint = getFrameOrigin();
-  auto frameSize = getFrameSize();
-  SkRect rect = SkRect::MakeXYWH(framePoint.x,framePoint.y,frameSize.width,frameSize.height);
-  canvas->drawRect(rect, paint);
+ 
+  /* apply view style props */
+  auto borderMetrics=viewProps.resolveBorderMetrics(component.layoutMetrics);
```

Comment:
here on usage of layoutMetrics : 
      layoutMetrics.frame.origin => relative (x,y) value 
      getFrameOrigin()  =>  absolute(x,y) value